ping-payload-extensions/extensions/template.md

```diff
@@ -0,0 +1,25 @@
+# [Title]
+
+[Description]
+
+## version [Version Number]
+
+[Version Description]
+
+Ping payload
+```python
+[Container Name] = Container(
+  type: [Type Number],
+  version: [Type Version],
```

Comment:
These feel potentially redundant.  Seems like we could simply have a `type` and if we end up needing to upgrade one of these types, we can use a new type byte.

Reply:
sure that is fine